VERSIONS

```diff
@@ -40,7 +40,7 @@ PXC80_INNODB="24"
 PXC80_REV="2119e75"
 PXC80_WSREP="4.14(779b689)"
 PROXYSQL_VER="1.4.16-percona-1.1"
-PROXYSQL2_VER="2.5.2-percona-1.1"
+PROXYSQL2_VER="2.5.3-percona-1.1"
```

Comment:
Please confirm with Vadim why we dont see percona in the version name now
Output:
proxysql --version
ProxySQL version 2.5.3-89-g86ce115, codename Truls

Reply:
ok

Reply:
Issue is fixed 
```
puneet@puneet-yoga-slim-7-carbon-13itl5:~$ proxysql --version
ProxySQL version 2.5.3-percona-1.1, codename Truls
```